src/features/restaurants/Dashboard.tsx

```diff
@@ -11,12 +11,14 @@ import { Error } from '../../components/Error'
 import { useNavigate } from 'react-router-dom'
 import { APP_ROUTES } from '../../utilities/constants'
 import { Button } from 'react-bootstrap'
+import { getLifts } from '../lifts/liftsSlice' 
 
 export const Main = () => {
   const dispatch = useDispatch<AppDispatch>()
   const navigate = useNavigate()
-  const { status, error } = useSelector((state: RootState) => state.restaurants)
+  const { status } = useSelector((state: RootState) => state.restaurants)
```

Comment:
@Nagel29 when you get a chance could you update `line 16` to be `Dashboard` instead of `Main`? Sorry, I forgot to change that when I renamed the file!

Reply:
No problem, it's updated now!

Reply:
Thank you!